lighthouse-core/gather/driver.js

```diff
@@ -833,6 +833,16 @@ class Driver {
     const promiseArr = urlPatterns.map(url => this.sendCommand('Network.addBlockedURL', {url}));
     return Promise.all(promiseArr);
   }
+
+  handleJavaScriptDialogs() {
```

Comment:
You'll need to put the enable in the promise chain:

```javascript
handleJavaScriptDialogs() {
  this.sendCommand('Page.enable').then(_ => {
    this.on('Page.javascriptDialogOpening', _ => {
      this.sendCommand('Page.handleJavaScriptDialog', {
        accept: true,
        promptText: 'Lighthouse prompt response',
      });
    });
  });
}
```

Can you also add a doc string to this method?